external/fv3fit/tests/test__config.py

```diff
@@ -61,20 +41,3 @@ def test_safe_dump_training_config():
             yaml.safe_dump(as_dict, f)
         from_dict = TrainingConfig.from_dict(as_dict)
         assert config == from_dict
-
-
-@pytest.mark.parametrize(
```

Comment:
Yes, I'll update the docs above. What I'd really like to do is remove the `additional_variables` field entirely, and instead automatically determine them based on other configuration settings (e.g. whether a derived variable is present in inputs) the way @AnnaKwa did for the offline diags, but that would be a longer task.

Reply:
Updated, though it's really not great that RandomForestHyperparameters has to reference the TrainingConfig in this way. In the future we can add a `additional_variables` property to Hyperparameters classes which returns the variable names needed. Then this can be done programmatically, without having a hard-code at the top level for e.g. `scaler_type` implying a certain value.